FireRender.Maya.Src/FireRenderExportCmd.cpp

```diff
@@ -550,10 +558,21 @@ MStatus FireRenderExportCmd::doIt(const MArgList & args)
 			if (statusExport != RPR_SUCCESS)
 			{
 				MGlobal::displayError("Unable to export fire render scene\n");
+				if (rprsContext != nullptr)
```

Comment:
please consider using context from frwrap to make it destroyed automatically. Problem will happen if exception will be thrown. In this case context will not be destroyed